src/app/space/add-app-overlay/add-app-overlay.component.spec.ts

```diff
@@ -60,8 +60,8 @@ describe('AddAppOverlayComponent', () => {
       });
     },
     validateProjectName(projectName: string): boolean {
-      // allows only '-', '_', ' ' and 4-40 characters (must start and end with alphanumeric)
-      const pattern = /^[a-zA-Z0-9][a-zA-Z0-9-_\s]{2,38}[a-zA-Z0-9]$/;
+      // allows only '-', ' ' and 4-40 characters (must start and end with alphanumeric)
+      const pattern = /^[a-zA-Z0-9][a-zA-Z0-9-\s]{2,38}[a-zA-Z0-9]$/;
```

Comment:
The spec does not allow for upper case letters either. Please update accordingly.

Reply:
Updated the regex here https://github.com/fabric8-ui/fabric8-ui/pull/3407/commits/0cf747190aff413b812511016bb306e0bc156d7b.